Ladybird/Android/src/main/res/layout/activity_main.xml

```diff
@@ -16,7 +16,22 @@
             android:id="@+id/toolbar"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            app:layout_scrollFlags="scroll|snap|enterAlways"/>
+            app:layout_scrollFlags="scroll|snap|enterAlways">
+
+            <EditText
```

Comment:
I think this also wants `android:inputType="textUri"`

Reply:
It has that below

Reply:
:man_facepalming: I don't know why I couldn't see that.